executor/pom.xml

```diff
@@ -30,6 +30,11 @@
       <groupId>com.redhat.service.bridge</groupId>
       <artifactId>infra</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-dmn-feel</artifactId>
+      <version>7.57.0.Final</version>
```

Comment:
Move version to depMan section.
Btw we can also consider to import `io.quarkus.platform:quarkus-kogito-bom` so that this dependency should be declared as transitive dependency